synthtool/gcp/templates/python_library/.kokoro/release.sh

```diff
@@ -26,7 +26,7 @@ python3 -m pip install --upgrade twine wheel setuptools
 export PYTHONUNBUFFERED=1
 
 # Move into the package, build the distribution and upload.
-TWINE_PASSWORD=$(cat "${KOKORO_KEYSTORE_DIR}/73713_google_cloud_pypi_password")
+TWINE_PASSWORD=$(cat "${KOKORO_GFILE_DIR}/secret_manager/google-cloud-pypi-password")
```

Comment:
nit: name the secret `-token` instead of `-password` if it's a token?

Reply:
SGTM, I'll create a new secret with that name.